docs/monitoring/monitoring.md

```diff
@@ -1,241 +1,237 @@
 # SnappyData Pulse
 
-SnappyData Pulse is a monitoring system that gives you a high-level overview of the status and performance of the cluster. It provides a simple widget based view, which helps you easily navigate and monitor your cluster.</br>
+SnappyData Pulse is a monitoring system that gives a high-level overview of the status and performance of the cluster.  Pulse provides a simple widget based view, which allows you to easily navigate, visualize, and monitor your cluster. You can monitor the overall status of the cluster as well as the status of each member in the cluster. 
+All the usage details are automatically refreshed after every 5 seconds, thus you can supervise the live status of your cluster.
 
-To access the SnappyData Pulse, start your cluster and open http:`<leadhost>`:5050/dashboard/ in your web browser. </br>
-`<leadhost>` is the hostname or IP of the lead node in your cluster.
 
-![Dashboard](../Images/monitoring_topnav.png)
+To access SnappyData Pulse, start your cluster and open [http:`<leadhost>`:5050/dashboard/](http:`<leadhost>`:5050/dashboard/) in your web browser.
 
-The top-right side of the page displays the date and time when the Dashboard was last updated. Click on the product version number to view details like the build number, source revision, underlying spark version, etc.
+!!!Note
+	`<leadhost>` is the hostname or IP of the lead node in your cluster.
+
+![Dashboard](../Images/Dashboard.png)
 
 !!! Note
-	
-	When using Smart Connector with upstream Spark, the **Dashboard** and **Member Details** sections are not displayed. Only the **SQL**, **Jobs** and **Stages** related information is displayed.
+
+	When using Smart Connector with upstream Spark, the **Dashboard** and **Member Details** sections are not displayed. Only the information related to **SQL**, **Jobs**, and **Stages** is displayed.
 
 The following topics are covered in this section:
```

Comment:
Order of topics and the order of actual sections is not matching.
Member Details View link is missing.